Core/Core/Login/LoginFindSchoolViewController.swift

```diff
@@ -146,6 +146,7 @@ class LoginFindSchoolViewController: UIViewController {
             host = "\(host).instructure.com"
         }
         searchField.resignFirstResponder()
+        env.lastLoginAccount = APIAccountResult(name: "", domain: host, authentication_provider: nil)
```

Comment:
Is this doing anything? If I don't log in just go back from the login screen to the welcome page I don't see any change. Should we update this `lastLoginAccount` field at all prior to logging in?

Reply:
We need to pass the selected account, then save it after login.